.travis.yml

```diff
@@ -17,7 +17,8 @@ env:
   - USR_AUTH_AUDIENCE="https://dev.data.humancellatlas.org/"
   - SVC_AUTH_AUDIENCE="https://dev.data.humancellatlas.org/"
   - AUTH_ISSUER="https://humancellatlas.auth0.com/"
-  - SECURITY_WHITELIST_DOMAINS="ebi.ac.uk"
```

Comment:
Just to confirm: Do we no longer need the `SECURITY_WHITELIST_DOMAINS` environment variable?

Reply:
yup, checking the group claim in the token is set to `hca` should be sufficient. It wouldn't get set as `hca` if it doesn't belong to the whitelist. The whitelist is being maintained in the HCA Auth0 account which Fusillade/DCP Auth Service is using for authentication

Reply:
I presume this would have been wrong anyway since domains other than `ebi.ac.uk` should be whitelisted? e.g. broad, czi, etc.

Reply:
yeah, and there might be more. The advantage of checking the group claim in the token is that we won't have to maintain that whitelist as well.